### ISSUE TITLE
Set up Google Voice and e-mail address (gmail?) for food coordination

### ISSUE THREAD

**Issue by ErnieAtLYD** [Code-for-Miami/i-have-food-i-need-food#2]
From Dutch Small:

- [ ] Set up a Google Voice number that we will use for dispatch. That way, there can be a transcript that is easily forwarded wherever it needs to go. The outgoing message should say something akin to "You have successfully reached Miami Food Dispatch. Your message will be transcribed and sent to our team. Please leave your name, your phone number, the number of meals needed, and the address of your location. We will help as quickly as we can. If you are able, please submit your request to www.thenameofyourwebsite.com." 
- [ ] Then, set up an email address that dispatch will continuously check. Provide @ernie with the login address and password to forward to Dutch.

**Comment by ernie** [comment]
Don't provide @ernie with the login, because he's not the Ernie you're looking for. Thanks. <3

**Comment by ErnieAtLYD** [comment]
@ernie sorry about that, and thanks for being not terrible about it ;)

**Comment by CyberStrike** [comment]
@ErnieAtLYD Is this still a thing that needs to happen?

**Comment by ErnieAtLYD** [comment]
No. I ended up purchasing a twilio phone number - 786-686-0665. I also created a GMail address, miamiemergencyfoodexchange@gmail.com, as a way to have a consolated account for everyone to use. I need to figure out how to get that password to you not through this github thread.

**Comment by michellebonat** [comment]
@ErnieAtLYD comment above noted, shall I share the google have food/want food forms I created for issue https://github.com/Code-for-Miami/Miami-Emergency-Food-Exchange/issues/3 to the new admin email address as well? Note that email as written above may have a typo.
